server/utils/dateTime.js

```diff
@@ -0,0 +1,28 @@
+const moment = require("moment-timezone");
+
+// Converts a time to a date object
+module.exports.convertToDate = (time, day, format) => {
+  let now = new Date();
+  if (format === "h:m") {
+    now.setHours(time.substr(0, time.indexOf(":")));
+    now.setMinutes(time.substr(time.indexOf(":") + 1));
+    now.setDate(day);
+    now.setSeconds(0);
+    return now;
+  } else return "Invalid Format";
+};
```

Comment:
Interesting approach, is there possibly another way this can be done? If we utilize Dates more? Maybe with epochs? Just opening this up for discussion

Reply:
I think so, I went with this approach because I used Material-UI's time picker component and it returns the time as 17:00, 20:00, etc and this was the only way I could convert only hours and minutes to a date string but I'll look into the other approaches you suggested. I'm interested in finding a much more cleaner solution.